LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2596,6 +2596,43 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chipset Fan", 3));
                         break;
 
+                    case Model.Z690_Extreme:
+                        v.Add(new Voltage("Vcore", 0, 10, 10));
+                        v.Add(new Voltage("+5V", 1, 20, 10));
+                        v.Add(new Voltage("AVCC", 2, 34, 34));
+                        v.Add(new Voltage("+3.3V", 3, 34, 34));
+                        v.Add(new Voltage("+12V", 4, 110, 10));
+                        v.Add(new Voltage("CPU Input Voltage", 5, 1, 1));
+                        v.Add(new Voltage("DRAM", 6));
+                        v.Add(new Voltage("+3.3V Standby", 7, 34, 34));
+                        v.Add(new Voltage("VBat", 8, 34, 34));
+                        v.Add(new Voltage("VTT", 9));
+                        v.Add(new Voltage("1.05V CPU Voltage", 10, 1, 1));
+                        v.Add(new Voltage("0.82V Chipset Voltage", 11, 1, 1));
+                        v.Add(new Voltage("1.0V Chipset Voltage", 12));
+                        v.Add(new Voltage("CPU SA", 13, 1, 1));
```

Comment:
Would you rather keep CPU SA, or have CPU System Agent be the name of this voltage name?

I've seen CPU SA be the more common name, so I'm assuming this would be preferred for consistency reasons, but let me know if you suggest otherwise.

Reply:
CPU System Agent is fine, you can rename all other "CPU SA" too if you'd like.

Reply:
I think it might be more ideal to rename other instances of CPU SA and other names in a separate PR for now - that way this PR remains in scope of what it's trying to achieve. Once this is approved and merge, I can look into renaming for other boards